en/integrations/plugin-packs/tutorials/troubleshooting-plugins.md

```diff
@@ -0,0 +1,255 @@
+`--proxyurl='<proto>://<proxy_addr>:<proxy_port>`. 
+
+#### UNKNOWN: 500 Can't connect to <ip_address>:<port> (Connection refused)
+
+This issue generally means that the port or protocol used by the Plugin is wrong, 
+misconfigured, or unsupported. 
+
+In this situation, at the Host configuration level, double-check that:
+* the port used is correct, primarily if you use a non-standard port for security reasons
+* the protocol (http or https) used matches the one configured on the API-side
+
+Each Plugin using HTTP backends does have `--proto` and `--port` options allowing 
+you to specify these values.
+
+#### UNKNOWN: 500 Can't connect to <ip_address>:<port> (Timeout)
+
+The timeout error occurs when the Plugin doesn't succeed in contacting the server 
+or that a third-party device is blocking or filtering the client's request. 
+
+#### UNKNOWN: 500 Can't connect to <ip_address>:<port> (<SSL Error>)
+
+SSL Errors indicate that the Plugin has some trouble establishing a secure connection 
+to get the monitoring information.
+
+The primary cause could be the certificate used. In this case, the best practice 
+would be either to: 
+* renew the certificate when it expired 
+* sign the remote certificate officially
+* deploy the certificate locally so the Plugin can recognize it
+
+Regardless of what HTTP backend you're using, it's possible to ignore SSL certificate 
+errors by adding specific flags: 
+
+* lwp backend: `--ssl-opt='--ssl-opt="SSL_verify_mode => SSL_VERIFY_NONE'`
+* curl backend: `--curl-opt='CURLOPT_SSL_VERIFYPEER => 0'`
+
+Sometimes, the remote host doesn't support negotiation about the SSL implementation, 
+so you must specify explicitly which one the Plugin has to use thanks to the `--ssl` 
+option (e.g. `--ssl='tlsv1'`). Refer to the manufacturer of software editor documentation.
+
+## SSH and CLI checks
+
+### UNKNOWN: Command error: <interpreter>: <command_name>: command not found
+
+This error warns that the Plugin is not able to execute the <command_name> because it 
+doesn't exist in PATH or is not installed.
+
+Depending on how the check is performed (locally or remotely), make sure that the 
+utility the plugin uses is available to your monitoring user. 
+
+### UNKNOWN: Command error: Host key verification failed.
+
+SSH-Based checks can use several backends. Either you use the `ssh` or `plink` backend, 
+you have to manually validate the remote system fingerprint from the centreon-engine 
+user on the monitoring Poller. If you don't do that, the Plugin will hang and timeout
+because it cannot accept the fingerprint for obvious security reasons.
+
+## NRPE checks
+
+### CHECK_NRPE STATE (CRITICAL|UNKNOWN): Socket timeout after 10 seconds
+
+Here are the questions you may want to ask yourself when obtaining this result: 
+
+- Does my IP Address and port parameter are correct? 
+- Is the NRPE daemon running on the remote system?
+- Is there any firewall or security policy that might block the request? 
```

Comment:
Maybe we should stick to a Markdown standard for unordered lists ? Either '*' or '-' but always use the same, what do you think ?
(https://www.markdownguide.org/basic-syntax/#unordered-list-best-practices)

Reply:
Yes, you're definitely right.